### ISSUE TITLE
Tensorflow frame segmentation

### ISSUE THREAD

**Issue by jimde** [OttaboticsRobotRacing/MK2#43]
Convert an existing Tensorflow (python) project which trains a neural network to segment the road from the background to c++.

The script to train the network can be kept in Python. However, it will need to be modified to save the model using protobuf after it has been trained.

The trained model will then be loaded into a c++ program which parses images and generate a mask for the road.
